assets/ui/debug.ui

```diff
@@ -0,0 +1,4 @@
+{
+  "type": "FlexibleMovementDebugLayout",
```

Comment:
The code of this class is commented out. Either one the first things to revive, or to throw out in a follow up and write down an issue for instead.